README.md

```diff
@@ -18,3 +41,13 @@ deploy desired stack
 ```sh
 ./infra.sh test-env test-stack
 ```
+
+
+## Deployment options
+
+You may add some extra options to `infra.sh`:
+
+* **-e redeploy_all=yes** - Apply the playbook to existing VMs once
```

Comment:
with this options all stack`s VMs will be recreated from scratch (destroy VM -> create VM), not just reapply playbooks.